bedrock/firefox/templates/firefox/features/adblocker.html

```diff
@@ -0,0 +1,216 @@
+     {% endif %}
+     {% if meta %}
+       <p class="mzp-c-card-meta">{{ meta }}</p>
+     {% endif %}
+     {% if cta %}
+       <p class="mzp-c-card-cta"><span class="mzp-c-card-cta-text">{{ cta }}</span></p>
+     {% endif %}
+     </div>
+   </a>
+   {% if video_url %}
+   <div class="mzp-c-card-video-wrapper hidden">
+     <figure class="mzp-c-card-video-content">
+       <video class="mzp-c-card-video" preload="none" controls playsinline data-ga-label="{{ ga_title }}">
+         <source src="{{ video_url }}.webm" type="video/webm">
+         <source src="{{ video_url }}.mp4" type="video/mp4">
+       </video>
+       <figcaption>
+         <p>{% if desc %}{{ desc }}{% endif %} <a href="{{ link_url }}">{{ _('Read more') }}</a></p>
+       </figcaption>
+     </figure>
+   </div>
+   {% endif %}
+ </section>
+
+ {%- endmacro %}
+
+ {% block content %}
+
+  <section class="mzp-c-hero mzp-t-product-firefox">
+    <div class="mzp-l-content">
+      <div class="mzp-c-hero-body">
+        <h1 class="mzp-c-hero-title">{{ _('Everything you need to know about ad blockers.') }}</h1>
+        <div class="mzp-c-hero-desc">
+          <p>{{ _('Ad blocker is a web browser extension-- that can detect web ads and prevent them from being displayed on a web page.') }}</p>
+        </div>
+      </div>
+    </div>
+  </section>
+
+  <section class="mzp-c-card-feature mzp-has-aspect-3-2 mzp-l-card-feature">
+    <div class="mzp-c-card-feature-media-wrapper">
+      {{ high_res_img('firefox/features/privacy-illustration1.png', {'alt': '', 'class': 'mzp-c-billboard-image'}) }}
+    </div>
+    <h1 class="mzp-c-hero-title">{{ _('How ad blockers work') }}</h1>
+    <div class="mzp-c-card-feature-content">
+      <div class="mzp-l-content mzp-l-card-half">
+        <div class="mzp-c-card">
+          <h2 class="mzp-c-card-feature-title">{{ _('A typical ad blocker') }}</h2>
+          <p>{{ _('Most ad blockers work in two ways.') }}</p>
+          <p>{{ _('First, they block communication from ad servers or others. Ads simply can’t get through.') }}</p>
+          <p>{{ _('Second, they hide parts of a web page that could be ads-- any section of a site that’s labeled "ad," for example. Most ad blockers use public and up-to-date lists of known ad servers.') }}</p>
+        </div>
+
+        <div class="mzp-c-card">
+          <h2 class="mzp-c-card-feature-title">{{ _('The Firefox ad blocker') }}</h2>
+          <p>{{ _('Firefox doesn’t include ad blocking out of the box, but it does include <a href="https://blog.mozilla.org/firefox/tracking-protection-always-on/">Tracking Protection</a>, which stops sites from installing those tracking cookies we talked about.') }}</p>
+          <p>It’s <a href="https://support.mozilla.org/en-US/kb/tracking-protection">easy to turn on</a>, and has the added benefit of making Firefox significantly faster.</p>
+        </div>
+      </div>
+    </div>
+  </section>
+
+  <section class="mzp-c-hero">
```

Comment:
I'm not sure this really needs to be a hero component, as it's just a headline with copy.